alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultProviderDataAccessor.java

```diff
@@ -73,7 +73,7 @@ public DefaultProviderDataAccessor(ProviderProjectRepository providerProjectRepo
         this.configurationAccessor = configurationAccessor;
     }
 
-    @Override
+    //    @Override
```

Comment:
We should delete all of these commented annotations and worry about re-adding them if we ever want to implement the interface again.